src/main/java/org/folio/entlinks/service/links/AuthorityDataStatService.java

```diff
@@ -0,0 +1,37 @@
+package org.folio.entlinks.service.links;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.folio.entlinks.domain.entity.AuthorityDataStat;
+import org.folio.entlinks.domain.entity.AuthorityDataStatStatus;
+import org.folio.entlinks.domain.repository.AuthorityDataStatRepository;
+import org.springframework.stereotype.Service;
+
+@Log4j2
+@Service
+@RequiredArgsConstructor
+public class AuthorityDataStatService {
+
+  private final AuthorityDataStatRepository statRepository;
+
+  private final AuthorityDataService authorityDataService;
+
+  public List<AuthorityDataStat> createInBatch(List<AuthorityDataStat> stats) {
+    var authorityDataSet = stats.stream()
+      .map(AuthorityDataStat::getAuthorityData)
+      .collect(Collectors.toSet());
```

Comment:
we can use `.toSet()` method instead of `.collect(Collectors.toSet())`

Reply:
There is no such method
![image](https://user-images.githubusercontent.com/5904625/213161212-df9e9b6b-cd98-46b1-b5cd-50e27f4f6df5.png)